UPGRADE-GUIDE.md

```diff
@@ -0,0 +1,1298 @@
+  - The `:hover`, `:active`, `:focus`, and `:disabled` selectors still have styles applied to them
+
+Renamed variables:
+- `--pf-c-app-launcher--m-top__menu--Transform` to `--pf-c-app-launcher--m-top__menu--TranslateY`
+- `--pf-c-app-launcher__menu-item-external-icon--Transform` to `--pf-c-app-launcher__menu-item-external-icon--TranslateY`
+
+Removed variables:
+- `--pf-c-app-launcher__group--PaddingTop`
+- `--pf-c-app-launcher__group--first-child--PaddingTop`
+- `--pf-c-app-launcher__menu--BorderWidth`
+- `--pf-c-app-launcher__separator--Height`
+- `--pf-c-app-launcher__separator--BackgroundColor`
+- `--pf-c-app-launcher__separator--MarginTop`
+- `--pf-c-app-launcher__separator--MarginBottom`
+- `--pf-c-app-launcher__separator--Height`
+
+### Backdrop
+- Removed blur [(#3009)](https://github.com/patternfly/patternfly/pull/3009)
+
+Renamed variables:
+- `--pf-c-backdrop--Color` to `--pf-c-backdrop--BackgroundColor`
+
+Removed variables:
+- `--pf-c-backdrop--BackdropFilter`
+
+### Breadcrumb
+- Added word-break, display link and icon inline, move divider [(#2916)](https://github.com/patternfly/patternfly/pull/2916)
+- Removed hover, active, focus, and disabled from workspace [(#2975)](https://github.com/patternfly/patternfly/pull/2975)
+- Made redhat font default, add opt-in for overpass [(#2955)](https://github.com/patternfly/patternfly/pull/2955)
+
+Updated markup:
+- Any reference to the divider element `.pf-c-breadcrumb__item-divider` will need to be moved in the markup to come before the breadcrumb link element `.pf-c-breadcrumb__link` instead of after it.
+
+Updated CSS:
+- The text will now break if there is a long string in an item link that is wider than the viewport.
+- The divider will always appear inline with the item text instead of wrapping to a new line in some cases.
+
+Removed classes:
+- `.pf-m-hover` from `.pf-c-breadcrumb__link`
+  - The `:hover` selector still has styles applied to it.
+
+Renamed variables:
+- `--pf-c-breadcrumb__item-divider--MarginLeft` to `--pf-c-breadcrumb__item-divider--MarginRight`
+
+Removed variables:
+- `--pf-c-breadcrumb__item--FontWeight`
+- `--pf-c-breadcrumb__link--FontWeight`
+
+### Button
+- Removed hover, active, focus, and disabled from workspace [(#2975)](https://github.com/patternfly/patternfly/pull/2975)
+- Cleaned up vars [(#3028)](https://github.com/patternfly/patternfly/pull/3028)
+- Updated control variation disabled state [(#3049)](https://github.com/patternfly/patternfly/pull/3049)
+- Added modifiers to position icon in button [(#2828)](https://github.com/patternfly/patternfly/pull/2828)
+- Made shield styles optional by default [(#2872)](https://github.com/patternfly/patternfly/pull/2872)
+
+Updated markup:
+- Adds the modifiers `.pf-m-start` and `.pf-m-end` to position `pf-c-button__icon` in the button component depending on whether it comes before or after text. If you are using a button with an icon, you will need to use one of these modifiers to style the icon.
```

Comment:
```suggestion
- Adds the modifiers `.pf-m-start` and `.pf-m-end` to position `pf-c-button__icon` in the button component depending on whether it comes before or after text. If you are using a button with an icon, use one of these modifiers to style the icon.
``` 
Is "Adds" supposed to be "Added" ?

Reply:
Yes!

Reply:
yep. A lot of these blurbs probably came from PR description notes written like "This PR adds the modifiers..." But for consistency in tense in the release notes, they should be updated.